segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/DebugBookKeeperLogWrapper.java

```diff
@@ -264,6 +264,19 @@ public void deleteDurableLogMetadata() throws DurableDataLogException {
         }
     }
 
+    public void deleteLedgersStartingWithId(long startId) throws DurableDataLogException {
```

Comment:
what is the purpose of this method @kuldeepk3 ?

Reply:
this method will delete all the ledgers >=  startId ( this is user input)